src/Illuminate/Database/Eloquent/Model.php

```diff
@@ -356,9 +356,10 @@ public function getGlobalScopes()
 	 * Register an observer with the Model.
 	 *
 	 * @param  object|string  $class
+	 * @param  int            $priority
```

Comment:
> I just checked the syntax in other docblocks and they are aligned as well

Yeh, some of our code is like that, but our rule for all new code is don't align the phpdoc.